src/view/com/post-thread/PostThread.tsx

```diff
@@ -507,12 +490,15 @@ function* flattenThreadSkeleton(
   node: ThreadNode,
   hasSession: boolean,
   treeView: boolean,
+  isTraversingReplies: boolean = false,
 ): Generator<YieldedItem, void> {
   if (node.type === 'post') {
-    if (node.parent) {
-      yield* flattenThreadSkeleton(node.parent, hasSession, treeView)
-    } else if (node.ctx.isParentLoading) {
-      yield PARENT_SPINNER
+    if (!node.ctx.isParentLoading) {
+      if (node.parent) {
+        yield* flattenThreadSkeleton(node.parent, hasSession, treeView, false)
+      } else if (!isTraversingReplies) {
+        yield TOP_COMPONENT
```

Comment:
Note this is now rendered only when the parent tree has loaded. This prevents jumps.